lib/parsers/asm-parser.ts

```diff
@@ -24,11 +24,58 @@
 
```

Comment:
Cool! Another js bites the dust :) Mayybe another thing that could be a separate change? (if there's nothing cc65 particular in this?)

Reply:
I'm going to merge as is. I have confidence in it working in its current form because of the unittests, and it takes a lot of time to separate properly without making mistakes. I'll remember to do things separately next time.